src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -126,6 +126,10 @@ class DocBaseViewer extends BaseViewer {
         this.docEl.classList.add('bp-doc');
         this.docEl.tabIndex = '0';
 
+        if (this.options.enableAnnotationsDiscoverability) {
+            this.docEl.classList.add('ba-discoverability-enabled');
```

Comment:
I wonder if this class should be named something else, like `bp-annotations-discoverable`? We don't reference any `.ba-*` classes in preview anywhere. Thoughts?

Reply:
Yes I think this makes sense.